interfaces/HO-Portal/src/app/program/edit-person-affected-popup/edit-person-affected-popup.component.ts

```diff
@@ -136,4 +145,22 @@ export class EditPersonAffectedPopupComponent implements OnInit {
   public closeModal() {
     this.modalController.dismiss();
   }
+
+  private getFspList() {
+    this.fspList = [];
+
+    if (this.programId) {
```

Comment:
If you turn this around, into an "early return", it would save 1 level of indentation for the rest of the logic and that would help for readability/understandability etc..

It will also highlight that "nothing will happen without a ProgramId".

So:
```
if (!this.programId) {
  return;
}
```

Or you can move the check outside of the `getFspList`-method and do a check at the very start in the `ngOnInit`-method..